src/schema/v2/show.ts

```diff
@@ -56,6 +57,18 @@ const FollowArtistType = new GraphQLObjectType<any, ResolverContext>({
   }),
 })
 
+export const ExhibitionPeriodFormatType = new GraphQLEnumType({
+  name: "ExhibitionPeriodFormat",
+  values: {
+    SHORT: {
+      value: "SHORT",
```

Comment:
**suggestion**: Might be worth adding descriptions with an example formatted periods?

```typescript
{
  description: "Long formatted period e.g. February 25 – May 24, 2015"
}
```

```typescript
{
  description: "Short formatted period e.g. Feb 25 - May 24, 2015"
}
```